### ISSUE TITLE
Impossible to activate the plugin 

### ISSUE THREAD

**Issue by ELiporaci** [dethegeek/customfields#82]
Hello

Friends, I have a question at the moment of activating the plugins in the GLPI System does not appear anything I download the plugins by the page indicated by the GLPI and put it in the folder but when I enter the GLPI system I do not recognize the complements I hope for your help .

**Comment by btry** [comment]
Hi

Custom Fields is EOL for more than 1 years.  Please use the plugin Fields.

https://github.com/pluginsGLPI/fields

**Comment by ELiporaci** [comment]
Yes, I do everything that is on the web pages but when I open the GLPI it does not appear.

![glpi1](https://user-images.githubusercontent.com/29778831/28923631-b0c34734-782c-11e7-88f4-037548c0b312.PNG)
![glpi2](https://user-images.githubusercontent.com/29778831/28923632-b0ca9bba-782c-11e7-9793-736bdddb8afb.PNG)

**Comment by ELiporaci** [comment]
Change the permissions of the server so that they can run but when I open the glpi application I do not see any plugins to perform the installation

**Comment by btry** [comment]
Hi 

Please show the content of the folder **glpi/plugins/**

**Comment by ELiporaci** [comment]
This is what appears to me in the GLPI folder 
![glpi3](https://user-images.githubusercontent.com/29778831/28923831-66b68402-782d-11e7-9cdc-0bb59b10bce5.PNG)

**Comment by ELiporaci** [comment]
The GPI is installed on a server called CPANEL in a hosting

**Comment by btry** [comment]
Rename **fields-master** into **fields**

About OCS, check the official documentation. A plugin for GLPI is not an archive file. It has a folder structure similar to the one you'll find in Fields. 

If you need more help to install a plugin, the forum is a better place: http://forum.glpi-project.org/

**Comment by btry** [comment]
One more thing: never chmod 0777. revert to chmod 0644
